From 443edcb56955a916f81676c3e11fe8dbd296df58 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Wed, 2 Dec 2020 13:29:48 +0700
Subject: [PATCH 231/357] dt-bindings: arm: renesas: Add HopeRun RZ/G2M v3.0
 boards

This patch adds board HiHope RZ/G2M v3.0 (the main board, powered by
the R8A774A3) and board HiHope RZ/G2 EX (the expansion board
that sits on top of the HiHope RZ/G2M v3.0). Both boards are made
by Jiangsu HopeRun Software Co., Ltd. (a.k.a. HopeRun).
Also update for board HiHope RZ/G2M v1.3 with powered by the R8A774A1.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 Documentation/devicetree/bindings/arm/shmobile.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/arm/shmobile.txt b/Documentation/devicetree/bindings/arm/shmobile.txt
index 3eb9590..856b9d6 100644
--- a/Documentation/devicetree/bindings/arm/shmobile.txt
+++ b/Documentation/devicetree/bindings/arm/shmobile.txt
@@ -98,12 +98,15 @@ Boards:
   - HopeRun expansion board for HiHope RZ/G2 platforms
     compatible = "hoperun,hihope-rzg2h", "renesas,r8a774e1",
                  "hoperun,hihope-rzg2m", "renesas,r8a774a1",
+                 "hoperun,hihope-rzg2m", "renesas,r8a774a3",
                  "hoperun,hihope-rzg2n", "renesas,r8a774b1",
                  "hoperun,hihope-rzg2-ex"
   - HopeRun HiHope RZ/G2H platform
     compatible = "hoperun,hihope-rzg2h", "renesas,r8a774e1"
-  - HopeRun HiHope RZ/G2M platform
+  - HopeRun HiHope RZ/G2M v1.3 platform
     compatible = "hoperun,hihope-rzg2m", "renesas,r8a774a1"
+  - HopeRun HiHope RZ/G2M v3.0 platform
+    compatible = "hoperun,hihope-rzg2m", "renesas,r8a774a3"
   - HopeRun HiHope RZ/G2N platform
     compatible = "hoperun,hihope-rzg2n", "renesas,r8a774b1"
   - iWave Systems RZ/G1C Single Board Computer (iW-RainboW-G23S)
-- 
2.7.4

